Index: geoclue-2.0/src/gclue-modem-manager.c
===================================================================
--- geoclue-2.0.orig/src/gclue-modem-manager.c
+++ geoclue-2.0/src/gclue-modem-manager.c
@@ -277,6 +277,15 @@ is_location_3gpp_same (GClueModemManager
         mcc = mm_location_3gpp_get_mobile_country_code (priv->location_3gpp);
         mnc = mm_location_3gpp_get_mobile_network_code (priv->location_3gpp);
         lac = mm_location_3gpp_get_location_area_code (priv->location_3gpp);
+
+        // Most likely this is an LTE connection and with the mozilla
+        // services they use the tracking area code in place of the
+        // location area code in this case.
+        // https://ichnaea.readthedocs.io/en/latest/api/geolocate.html#cell-tower-fields
+        if (lac == 0x0 || lac == 0xFFFE) {
+                lac = mm_location_3gpp_get_tracking_area_code(priv->location_3gpp);
+        }
+
         cell_id = mm_location_3gpp_get_cell_id (priv->location_3gpp);
 
         return (mcc == new_mcc &&
@@ -316,6 +325,15 @@ on_get_3gpp_ready (GObject      *source_
         mcc = mm_location_3gpp_get_mobile_country_code (location_3gpp);
         mnc = mm_location_3gpp_get_mobile_network_code (location_3gpp);
         lac = mm_location_3gpp_get_location_area_code (location_3gpp);
+
+        // Most likely this is an LTE connection and with the mozilla
+        // services they use the tracking area code in place of the
+        // location area code in this case.
+        // https://ichnaea.readthedocs.io/en/latest/api/geolocate.html#cell-tower-fields
+        if (lac == 0x0 || lac == 0xFFFE) {
+                lac = mm_location_3gpp_get_tracking_area_code(location_3gpp);
+        }
+
         cell_id = mm_location_3gpp_get_cell_id (location_3gpp);
 
         if (is_location_3gpp_same (manager, mcc, mnc, lac, cell_id)) {
Index: geoclue-2.0/src/meson.build
===================================================================
--- geoclue-2.0.orig/src/meson.build
+++ geoclue-2.0/src/meson.build
@@ -32,7 +32,7 @@ sources += [ 'gclue-main.c',
              'gclue-location.h', 'gclue-location.c' ]
 
 if get_option('3g-source') or get_option('cdma-source') or get_option('modem-gps-source')
-    geoclue_deps += [ dependency('mm-glib', version: '>= 1.6') ]
+    geoclue_deps += [ dependency('mm-glib', version: '>= 1.10') ]
     sources += [ 'gclue-modem.c',
 				 'gclue-modem.h',
 				 'gclue-modem-manager.c',
