--- a/configure
+++ b/configure
@@ -11579,16 +11579,16 @@
     else
         if test -n "$PKG_CONFIG" && \
     { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"
-      NetworkManager libnm_glib
+      NetworkManager libnm-glib
    \""; } >&5
   ($PKG_CONFIG --exists --print-errors "
-      NetworkManager libnm_glib
+      NetworkManager libnm-glib
    ") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
   pkg_cv_NETWORK_MANAGER_CFLAGS=`$PKG_CONFIG --cflags "
-      NetworkManager libnm_glib
+      NetworkManager libnm-glib
    " 2>/dev/null`
 else
   pkg_failed=yes
@@ -11603,16 +11603,16 @@
     else
         if test -n "$PKG_CONFIG" && \
     { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"
-      NetworkManager libnm_glib
+      NetworkManager libnm-glib
    \""; } >&5
   ($PKG_CONFIG --exists --print-errors "
-      NetworkManager libnm_glib
+      NetworkManager libnm-glib
    ") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
   pkg_cv_NETWORK_MANAGER_LIBS=`$PKG_CONFIG --libs "
-      NetworkManager libnm_glib
+      NetworkManager libnm-glib
    " 2>/dev/null`
 else
   pkg_failed=yes
@@ -11633,11 +11633,11 @@
 fi
         if test $_pkg_short_errors_supported = yes; then
 	        NETWORK_MANAGER_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "
-      NetworkManager libnm_glib
+      NetworkManager libnm-glib
    "`
         else
 	        NETWORK_MANAGER_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "
-      NetworkManager libnm_glib
+      NetworkManager libnm-glib
    "`
         fi
 	# Put the nasty error message in config.log where it belongs
