Index: geoclue/src/master-provider.c
===================================================================
--- geoclue.orig/src/master-provider.c	2011-06-23 12:06:14.681991365 +0200
+++ geoclue/src/master-provider.c	2011-06-23 12:08:24.169986815 +0200
@@ -846,10 +846,6 @@
 static gboolean
 gc_master_provider_initialize (GcMasterProvider *provider)
 {
-	GcMasterProviderPrivate *priv;
-	 
-	priv = GET_PRIVATE (provider);
-	
 	if (!gc_master_provider_initialize_interfaces (provider)) {
 		return FALSE;
 	}
